payment/src/main/java/org/killbill/billing/payment/retry/BaseRetryService.java

```diff
@@ -91,6 +94,30 @@ public void stop() throws NoSuchNotificationQueue {
         }
     }
 
+    public List<NotificationEventWithMetadata<PaymentRetryNotificationKey>> getRetryAttempts(final UUID attemptId, final InternalCallContext context) {
```

Comment:
If this is just for testing, should we instead implement this in our tests?

Reply:
sure, i 'll remove it altogether as my test actually does not really need it since we already return the pending attempts in the Payment#attempts